prepare-incremental-build/get-commit-metadata.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# branch="${GITHUB_BASE_REF}"
+
+# if [ "${GITHUB_EVENT_NAME}" = 'push' ]; then
+#   branch="${GITHUB_REF_NAME}"
+# fi
+
```

Comment:
maybe remove this now that this is handled in one of the action'step.

Reply:
done in my fork